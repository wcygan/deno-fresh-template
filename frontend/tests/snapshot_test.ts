import { assertSnapshot } from "jsr:@std/testing/snapshot";

const writeGranted = (await Deno.permissions.query({ name: "write" })).state ===
  "granted";

Deno.test({
  name: "hero fragment snapshot",
  ignore: !writeGranted,
  permissions: { read: true, write: true },
  async fn(t) {
    const html = "<section class=\"hero\">Welcome</section>";
    await assertSnapshot(t, html);
  },
});
