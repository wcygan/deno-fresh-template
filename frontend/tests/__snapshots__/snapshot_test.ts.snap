export const snapshot = {};

snapshot[`hero fragment snapshot 1`] = `'<section class="hero">Welcome</section>'`;
